src/plugins/vis_type_timeseries/public/application/components/color_rules.tsx

```diff
@@ -100,9 +120,14 @@ export class ColorRules extends Component<ColorRulesProps> {
 
   handleOperatorChange = (item: ColorRule) => {
     return (options: Array<EuiComboBoxOptionOption<string>>) => {
+      const selectedOperator: ColorRulesOperator | void = colorRulesOperatorsList.find(
```

Comment:
nit: `colorRulesOperatorsList` already typed. No need to add extra `ColorRulesOperator | void`

![image](https://user-images.githubusercontent.com/20072247/119118058-82c4d180-ba32-11eb-843e-d10d2711d603.png)